### PR TITLE
Remove all the different 'Organize Usings' commands and only offer 'Remove and Sort Usings/Imports'

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes https://github.com/dotnet/roslyn/issues/12031

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Tagging @dotnet/roslyn-ide

**Comment by CyrusNajmabadi** [comment]
test vsi please

**Comment by CyrusNajmabadi** [comment]
Feature looks like this now:

image

**Comment by CyrusNajmabadi** [comment]
test vsi please

**Comment by CyrusNajmabadi** [comment]
Closed side is here: https://github.com/dotnet/roslyn-internal/pull/1104

**Comment by CyrusNajmabadi** [comment]
test vsi please

**Comment by CyrusNajmabadi** [comment]
Ping @dotnet/roslyn-ide

File: src/VisualStudio/Core/Def/Commands.vsct [hunk 1]
```diff
@@ -474,6 +395,9 @@
     <KeyBinding guid="guidRoslynGrpId" id="cmdidGoToImplementation" mod1="Control" key1="VK_F12" editor="guidCSharpCodePageEditorFactory" />
     <KeyBinding guid="guidRoslynGrpId" id="cmdidGoToImplementation" mod1="Control" key1="VK_F12" editor="guidVisualBasicEditorFactory" />
     <KeyBinding guid="guidRoslynGrpId" id="cmdidGoToImplementation" mod1="Control" key1="VK_F12" editor="guidVisualBasicCodePageEditorFactory" />
+
+    <KeyBinding guid="guidCSharpGrpId" id="cmdidCSharpOrganizeRemoveAndSort" mod1="Control" key1="R" key2="G" editor="guidVSStd97" />
```
Comment (jasonmalinowski) [inline src/VisualStudio/Core/Def/Commands.vsct]: Might want to give a comment here about the ordering.

File: src/VisualStudio/Core/Def/Commands.vsct [hunk 2]
```diff
@@ -489,19 +413,9 @@
...
 
     <GuidSymbol name ="guidCSharpGrpId" value ="{5d7e7f65-a63f-46ee-84f1-990b2cab23f9}">
-      <IDSymbol name="IDM_CSHARP_ORGANIZE" value="0x3900"/>
-      <IDSymbol name="IDM_CSHARP_CTX_ORGANIZE" value="0x3950"/>
-
       <IDSymbol name="IDG_CSHARP_CODEGEN" value="0x3490"/>
-      <IDSymbol name="IDG_CSHARP_ORGANIZE" value="0x3617"/>
-      <IDSymbol name="IDG_CSHARP_CTX_ORGANIZE" value="0x3618"/>
 
-      <IDSymbol name="cmdidContextOrganizeRemoveUnusedUsings" value="0x1911"/>
```
Comment (jasonmalinowski) [inline src/VisualStudio/Core/Def/Commands.vsct]: I know I've seen extensions for VS in the past that would explicitly call through with these commands to programmatically do that. That was before we had a "fix all in solution" so it's possible we don't care, but you might want to do a quick search through the VS codebase to make sure nobody is relying on us still responding to these.

**Comment by jasonmalinowski** [comment]
:+1:, other than the broken internal build.

**Pull request merged**